gitlab-rocketchat.hooks.js

```diff
@@ -377,7 +377,8 @@ See: ${data.object_attributes.url}`
 		switch (event_name) {
 			case 'project_create':
 			case 'project_destroy':
-				action = event_name === 'project_create' ? 'created' : 'removed';
+			case 'project_update':
+				action = event_name === 'project_create' ? 'created' : (event_name === 'project_create' ? 'removed' : 'updated');
```

Comment:
I think it should be (event_name === 'project_destroy') here

Reply:
Indeed, was fixed in the last commit.